File: class03/cpp/AP0304-Exemplo02.cpp
#include <iostream>
#include <iomanip>
int main() {
    double valor, pago;
    std::cout << "Valor da compra: R$ "; std::cin >> valor;
    std::cout << "Valor pago: R$ "; std::cin >> pago;
    double troco = pago - valor;
    if (troco < 0) std::cout << "Dinheiro insuficiente!" << std::endl;
    else std::cout << "Troco: R$ " << std::fixed << std::setprecision(2) << troco << std::endl;
    return 0;
}

File: class03/python/AP0304-Exemplo02.py
# CalcularTroco — Aula 03
# Algoritmo de saque com cálculo de troco

valor = float(input("Valor da compra: R$ "))
pago = float(input("Valor pago: R$ "))
troco = pago - valor
if troco < 0:
    print("Dinheiro insuficiente!")
else:
    print(f"Troco: R$ {troco:.2f}")
